yt/utilities/command_line.py

```diff
@@ -932,6 +935,8 @@ def __call__(self, args):
                 plt.annotate_grids()
             if args.time:
                 plt.annotate_timestamp()
+            if args.scale:
```

Comment:
Does this need to be updated to `args.show-scale-bar`?

Reply:
Ooops, it does.

Reply:
That's fixed.